src/core/sys/windows/winnt.d

```diff
@@ -2451,27 +2456,65 @@ struct SECURITY_DESCRIPTOR {
     PACL Dacl;
 }
 alias SECURITY_DESCRIPTOR* PSECURITY_DESCRIPTOR, PISECURITY_DESCRIPTOR;
-
-enum TOKEN_INFORMATION_CLASS {
-    TokenUser = 1,
-    TokenGroups,
-    TokenPrivileges,
-    TokenOwner,
-    TokenPrimaryGroup,
-    TokenDefaultDacl,
-    TokenSource,
-    TokenType,
-    TokenImpersonationLevel,
-    TokenStatistics,
-    TokenRestrictedSids,
-    TokenSessionId,
-    TokenGroupsAndPrivileges,
-    TokenSessionReference,
-    TokenSandBoxInert,
-    TokenAuditPolicy,
-    TokenOrigin
+alias TOKEN_ELEVATION_TYPE = int;
+enum : TOKEN_ELEVATION_TYPE {
```

Comment:
Making this an anonymous enum breaks existing code.... Why?

Reply:
I was migrating from my own windows headers library and didn't know about this difference between them, thanks!